### PR TITLE
SP2-621 add e2e test for view/agree plan

### PR DESCRIPTION
**Pull request by pk2453**

### CODE REVIEW

File: integration_tests/e2e/plan-summary.cy.ts [hunk 1]
```diff
@@ -12,6 +12,71 @@ describe('View Plan Summary', () => {
...
+    cy.url().should('include', '/add-steps')
+  })
+
+  it('Plan with goals and steps should have required links and status as not started', () => {
+    cy.get<{ plan: PlanType }>('@plan').then(({ plan }) => {
+      cy.addGoalToPlan(plan.uuid, DataGenerator.generateGoalWithTitle('Test Accommodation'))
+      cy.visit('/plan-summary?source=nav')
+    })
+    cy.contains('a', 'Add steps').click()
+    cy.get(`#step-description-1-autocomplete`).type('Accommodation')
```
Comment (pipwilson) [inline integration_tests/e2e/plan-summary.cy.ts]: Are these supposed to be backticks or normal quote marks?

Comment (pk2453) [reply integration_tests/e2e/plan-summary.cy.ts]: No it suppose to be normal quote so updated now

Comment (pipwilson) [reply integration_tests/e2e/plan-summary.cy.ts]: great thanks

**Review by pipwilson — APPROVED** [review]
LGTM

**Pull request merged**